osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/server/Iec60870RtuSimulator.java

```diff
@@ -0,0 +1,36 @@
+package org.opensmartgridplatform.simulator.protocol.iec60870.server;
+
+import java.io.IOException;
+
+import org.openmuc.j60870.Server;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Iec60870RtuSimulator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Iec60870RtuSimulator.class);
+
+    private Iec60870ServerEventListener iec60870ServerEventListener;
+    private Server server;
+
+    public Iec60870RtuSimulator(final Iec60870ServerEventListener iec60870ServerEventListener) {
+        this.iec60870ServerEventListener = iec60870ServerEventListener;
+    }
+
+    public void start() {
+        this.server = new Server.Builder().build();
+
+        try {
+            LOGGER.info("Starting IEC60870 Server.");
+            this.server.start(this.iec60870ServerEventListener);
+        } catch (final IOException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1166)